front/src/modules/ui/board/components/BoardOptionsDropdownContent.tsx

```diff
@@ -85,6 +96,23 @@ export function BoardOptionsDropdownContent({
     onStageAdd?.(columnToCreate);
   };
 
+  const { upsertView } = useUpsertView({
+    onViewsChange,
+    scopeContext,
+  });
+
+  const handleViewNameSubmit = async () => {
```

Comment:
let's use function handleViewNameSubmit syntax per project guideline here. I thought it was in the linter but it doesn't seems to be!